config/routes.rb

```diff
@@ -35,66 +28,77 @@
   end
 
   scope "/shopowners", controller: :shops do
-    get "/shop/new" => :new,
-        as: :shop_new
-    get "/admin/shop/products" => :products,
-        as: :shop_products
-    get "/admin/dashboard" => :show,
-        as: :dashboard
-    post "/shops" => :create
-    get "/shops/:id/edit" => :edit, as: :edit_shop
+    get "/shop/new"            => :new, as: :shop_new
+    get "/admin/shop/products" => :products, as: :shop_products
+    get "/admin/dashboard"     => :show, as: :dashboard
+    post "/shops"              => :create
+    get "/shops/:id/edit"      => :edit, as: :edit_shop
   end
 
   scope "/shopowners", controller: :shop_owners do
-    get "/activate/:token" => :shop_owner_activate,
+    get "/activate/:token"   => :shop_owner_activate,
```

Comment:
Yes, I was trying to clean up the routes file and make it more readable. I ran rubocop though and it didn't raise it as a violation.